src/app/components/logged/balance/balance-transfer/balance-transfer.component.html

```diff
@@ -115,81 +126,60 @@ <h1 class="text-2xl font-bold">Transfeirir com Pix</h1>
         </button>
-        <button type="submit" class="buzzard-btn">Continuar com Pix</button>
+        <button type="submit" class="buzzard-btn w-1/2 font-bold">Continuar com Pix</button>
       </div>
     </form>
-    }
-  </div>
 
-  <div
-    class="w-1/2 flex flex-col items-center border-solid border-bzd-orange border-2 rounded-2xl p-3 h-fit"
-  >
-    <h1 class="text-2xl font-bold">Resumo da transferência</h1>
-    @if(!!balance){
-    <div class="w-full flex flex-col">
-      <p>Valor</p>
-      <div class="h-0.5 bg-bzd-orange w-full"></div>
-      <p class="text-2xl font-bold">R$ {{ balance | number : "1.2-2" }}</p>
-    </div>
+    <!-- Resumo da Transferência -->
+    <div
+      class="flex flex-col items-center border-2 border-bzd-orange rounded-2xl p-4 h-fit"
+    >
+      <h1 class="text-2xl font-bold mb-4">Resumo da transferência</h1>
+      <div [@ngIf]="balance" class="w-full flex flex-col mb-4 font-bold">
+        <p>Valor</p>
+        <div class="h-0.5 bg-bzd-orange w-full mb-2"></div>
+        <p class="text-2xl font-bold">R$ {{ balance | number : '1.2-2' }}</p>
+      </div>
 
-    } @if(!!transferType){
-    <div class="w-full flex flex-col">
-      <p>Forma de transferência</p>
-      <div class="h-0.5 bg-bzd-orange w-full"></div>
-      <p class="text-2xl font-bold">
-        {{ transferType == "agency" ? "Agência e Conta" : "Pix" }}
-      </p>
-    </div>
-    } @if(isTransferMethodSet){
-    <div class="w-full flex flex-col">
-      <p>Destino</p>
-      <div class="h-0.5 bg-bzd-orange w-full"></div>
+      <div [@ngIf]="transferType" class="w-full flex flex-col mb-4 font-bold">
+        <p>Forma de transferência</p>
+        <div class="h-0.5 bg-bzd-orange w-full mb-2"></div>
+        <p class="text-2xl font-bold">
+          {{ transferType == 'agency' ? 'Agência e Conta' : 'Pix' }}
+        </p>
+      </div>
 
-      @if(transferType == 'agency'){
-      <p class="">
-        Agência:
-        <span class="text-lg font-bold">
-          {{ agencyFormgroup.get("agency").value }}
-        </span>
-      </p>
-      <p class="">
-        Conta:
-        <span class="text-lg font-bold">
-          {{ agencyFormgroup.get("account").value }}
-        </span>
-      </p>
-      <p class="">
-        Banco:
-        <span class="text-lg font-bold">
-          {{ agencyFormgroup.get("bank").value }}
-        </span>
-      </p>
-      }@else {
-      <p class="">
-        Chave Pix:
-        <span class="text-lg font-bold">
-          {{ pixFormgroup.get("pix").value }}
-        </span>
-      </p>
-      }
-    </div>
-    }
+      <div [@ngIf]="isTransferMethodSet" class="w-full flex flex-col mb-4 font-bold">
+        <p>Destino</p>
+        <div class="h-0.5 bg-bzd-orange w-full mb-2"></div>
 
-    <button
-      type="button"
-      class="buzzard-btn"
-      [disabled]="!isTransferMethodSet"
-      (click)="submitOperation()"
-    >
-      Transfeirir
-    </button>
+        <div [@ngIf]="transferType == 'agency'">
```

Comment:
`[@ngIf]` não existe

Reply:
são todos os [@ngIf]? ou são esses que vc falou (linha 30, 54, 152, 164 e 169?

Reply:
se for todos os [@ngIf] qual outro comado devo utilizar?

Reply:
`@if(condição){código}`
O bloco de código fica dentro das chaves, basicamente é um if normal